### DISCUSSION TITLE
Did you experienced some issues with RobotDyn mega 2560 R3?

### DISCUSSION THREAD

**Discussion by littlebalup** [sanni/cartreader#97] [Need help]
Hi,
I recently made a pair of complete cartreader with the exact same components, including a pair of RobotDyn mega 2560 R3 programmed the exact same way.

I made them strictly following the instructions. Removing both the 5v regulator and the transitor from the dev boards,  adding the 470µf for SA1 chip, etc... 
The only thing I didn't installed yet is the rgb led (I need to order some translucid PLA first). But I doubt it could cause any issue.

One cartreader works great so far with all my games but I have an issue reading few original SNES cartridges on the other one. The rom is not detected at all with a "no header or corrupted header" error.
The strange thing is it works great with most of my cartridges (Street fighter 2, Illusion of time, secret of evermore, mechwarrior, secret of mana, super mario all stars, super punch out... and many more) but don't with few games like Zelda a link to the past (pcb SHVC-1A3B-13), Super mario World (pcb SHVC-1A1B-06) and Starwing (pcb SHVC-1C0N).

I interverted both mega 2560 between my both cartreaders to identify if the issue comes from my build or from the dev board. It looks like the mega 2560 is faulty as the other one works perfectly on both cartreaders.

So, I tried to identify what's could be wrong on the dev board without success. I checked every continuity between the atmega2560 chip and the socket header, checked if any shortcut, etc... It looks like the atmega2560 chip is faulty.
Did you experienced that kind of issue? Do you have any idea?

**Comment by sanni** [comment]
I once had an issue where an Arduino had a dead pin and didn't work right because of it and another time I had an issue with an Arduino acting weird until I disabled the brownout detection by flashing new fuse bits with an USBASP to the Atmega.
I do feel like the different Arduinos from different manufacturers behave slightly different too.

**Reply by littlebalup** [reply]
Thanks,
I ordered an USBASP, then I'll check and compare fuses between both. And try disabling BOD.
I also ordered another Mega 2560 r3 form another place. I'll see.
